### ISSUE TITLE
UI promotes downgrading version

### ISSUE THREAD

**Issue by cloo** [shaarli/Shaarli#1118]
https://i.imgur.com/vU1rGfh.png

**Comment by virtualtam** [comment]
Hi @cloo,

I see from #1117 that you are using Yunohost. AFAIK Shaarli is part of their application bundles, and is installed using the  [shaarli_ynh](https://github.com/YunoHost-Apps/shaarli_ynh) script.

Looking at how the scripts work, I see that:
- the version installed is `v0.9.0`, which is obsolete and contains several vulnerabilities
- a custom configuration is deployed, that slightly differs from the one supplied with Shaarli
- a custom Nginx configuration is used, which might not be up-to-date with Shaarli server requirements

I would suggest contacting the Yunohost team on the [shaarli_ynh](https://github.com/YunoHost-Apps/shaarli_ynh) tracker to address these issues.

Ping @Lapineige

---

*Here is what you would need to check on a vanilla installation:*

What is the value of `check_updates_branch` in `data/config.json.php`?

As `v0.9.x` is currently our `latest` branch, this configuration section should look like:

```json
    "updates": {
        "check_updates": true,
        "check_updates_branch": "latest",
        "check_updates_interval": 86400
    },
```

**Comment by cloo** [comment]
Thanks @virtualtam, appreciate your time! 
I've reported the issue to shaarli_ynh team - we'll see what that yields.

**Comment by lapineige** [comment]
Yes, this is a Yunohost packaging-related issue. The version is now updated.
So you can clearly close this issue 🙂
